src/views/Visualizer/View/View.js

```diff
@@ -343,10 +343,11 @@ export class Visualizer extends React.Component {
     if (!map) return;
 
     const getFilteredProperties = (properties, form) => (
-      Object.fromEntries(
-        Object.keys(properties)
-          .filter(prop => form.some(({ property }) => property ===  prop))
-          .flatMap(prop => Object.entries(getSearchParamFromProperty(properties, form, prop))),
+      Object.assign(
+        {},
+        ...Object.keys(properties)
+          .filter(propertyName => form.some(({ property }) => property === propertyName))
+          .map(propertyName => getSearchParamFromProperty(properties, form, propertyName)),
```

Comment:
Is there a difference with this? :

```suggestion
      {
        ...Object.keys(properties)
          .filter(propertyName => form.some(({ property }) => property === propertyName))
          .map(propertyName => getSearchParamFromProperty(properties, form, propertyName)),
      }
```

Reply:
Yes it is.

In the commit case, it's an **array's spread** so it returns all contained objects and the `Object.assign` will merge them all (Btw, the first `{}` of `Object.assign` is necessary in the case of the array is empty).

In the suggestion case, it's an **object's spread applied to an array**  so each item returns a key with the current index and a value with the expected object.